code/index.html

```diff
@@ -1,22 +1,209 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>(H)ear Mag</title>
+    <link rel="icon" type="image/png" href="./Ear.png" sizes="16x16 32x32" >
+    <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+   <header>
+      <img class="logo" img src="./Ear.jpg" alt="Ear drawing">
+      <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+      <h1 class="order-1">(H)ear Magazine</h1>
+      <nav class="order2">
+        <a href="https://www.instagram.com/devialet/?hl=en">INSTA</a>
+        <a href="https://www.facebook.com/UndentiaAB/">UNDENTIA</a>
+        <a href="https://www.ljudupplevelser.se/">STORE</a>
+      </nav>
     </header>
+  <main>
+    <section class="news-container">
+      <img src="./WeCanDoItB&W.jpg" alt="Listening tiger">
+      <div>
+      <h2>All you ever wanted to hear</h2>
+      <p>How connectivity and AI interplay to elevate the way we experience sound. <br>
+        Oh how the roar of a good sound system makes you purr with delight at newfound 
+        textures, depths and spaces in "old" music. The familiar scratch noise that gave 
+        character to old school records have been replaced with exceptional digital clarity. 
+        Technical wonders that send us to the moon and back. Soundscapes that bring us to the front-row. 
+        The singer is in your living room. You can hear each breath, each tongue click, the dryness of 
+        the mouth along with pure voice. And the band, oh the band is marching in too.</p>
+      </div>
+  </main>
+    <div class="container">
+      <div class="post-wrap">    
+      <div class="post">
+       <div class="post-front">
+       <div class="post-img" style="background-image: url(./Cat-crops-5.jpg);" alt="Falling Cat"></div>
+       <div class="post-info">
+       <span>Active Speakers</span>
+       <h2>Genelecly Sound</h2>
+       </div>
+       </div>
```

Comment:
Check the indentation so it will be easier to read and understand!

Reply:
Yes! I was confused by the guidelines where the shown example had aligned html aligned.